steps/apply/spec.schema.json

```diff
@@ -0,0 +1,94 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "type": "object",
+  "properties": {
+    "os": {
+      "type": "object",
+      "description": "Additional packages and commands to run prior to terraform execution",
+      "properties": {
+        "packages": {
+          "type": "array",
+          "description": "list of apk packages to install at container execution time"
+        },
+        "commands": {
+          "type": "array",
+          "description": "List of shell commands to run after packages are installed but before Terrafrom executes"
+        }
+      }
+    },
+    "workspace": {
+      "type": "string",
+      "description": "The name of the Terraform workspace to run (defaults to 'default')"
+    },
+    "credentials": {
+      "type": "string",
+      "description": "Relay secret containing the credentials.json for accessing GCP"
+    },
+    "aws": {
+      "type": "object",
```

Comment:
I think this can be `x-relay-connectionType: aws` right?

Reply:
Hm, last week i convinced myself it wouldn't work, because the connection doesn't have a `region` key which `ni aws` expected to see - but maybe that was wrong?

Reply:
Oh, yeah, this needs the `!Fn.merge` stuff. These steps all need to be updated at some point.